sys/src/9/amd64/io.h

```diff
@@ -279,6 +279,51 @@ struct Pcisiz
 	int	bar;
 };
 
+/*
```

Comment:
Same here. The name and date are not needed.
Also can you change the comment just a bit. 
Instead of "Add the definitions" just start with 
Definitions for the generic PCI capability structure based on